XamlControlsGallery/PageHeader.xaml

```diff
@@ -114,6 +115,38 @@
                         <AppBarButton.Icon>
                             <FontIcon Glyph="&#xE771;" Margin="0,-2,0,0" />
                         </AppBarButton.Icon>
+                        <AppBarButton.Resources>
+                            <controls:TeachingTip x:Name="ToggleThemeTeachingTip1"                   
+                            Target="{x:Bind ThemeButton}"
+                            Title="Change themes without hassle"                                        
+                            Subtitle="It's easier than ever to see control samples in both light and dark theme!">
+                                <controls:TeachingTip.IconSource>
+                                    <controls:SymbolIconSource Symbol="Refresh" />
+                                </controls:TeachingTip.IconSource>
+                            </controls:TeachingTip>
+                            <controls:TeachingTip x:Name="ToggleThemeTeachingTip2"                   
+                                Title="Change themes without hassle"
+                                Subtitle="It's easier than ever to see control samples in both light and dark theme!"
+                                PreferredPlacement="Auto"                                              
+                                PlacementMargin="20"
+                                IsLightDismissEnabled="True"
+                                ActionButtonClick="ToggleThemeTeachingTip2_ActionButtonClick"
+                                ActionButtonContent="Toggle theme now"
+                                CloseButtonContent="Got it!">
+                            </controls:TeachingTip>
+                            <controls:TeachingTip x:Name="ToggleThemeTeachingTip3" 
+                                Target="{x:Bind ThemeButton}"
+                                Title="Change themes without hassle"
+                                Subtitle="It's easier than ever to see control samples in both light and dark theme!"
+                                PreferredPlacement="LeftEdgeAlignedBottom">
+                                <controls:TeachingTip.HeroContent>
+                                    <Image Source="/Assets/SampleMedia/sunset.jpg" />
```

Comment:
Please give the image a UIA name.

Reply:
Resolved with [this commit.](https://github.com/Microsoft/Xaml-Controls-Gallery/pull/73/commits/c2c7e66d6822389dbf66ba63a73112ef6993a93e)